tmt/steps/prepare/brand_new_allmighty_install/providers/__init__.py

```diff
@@ -18,6 +18,7 @@
 class ArtifactType(enum.Enum):
     RPM = 'rpm'
     CONTAINER = 'container'
+    REPOSITORY_FILE = 'repository-file'
```

Comment:
Is repository file a type of an artifact? I thought artifacts are the bits to be tested (e.g. package) not the channel through which we get them (repo, repo file...).